CONTRIBUTORS

```diff
@@ -1,4 +1,5 @@
-Actual maintainer: Thomas Lecavelier (http://thomas.lecavelier.name)
```

Comment:
Looking at this, I feel we should just replace that file by a list. The hierarchy thing (*actual maintainer*) is weird and each time someone modifies that file I feel we leave someone out :(

Not that this is a blocker for the merge, just made me think of it.

More on this in #87

Reply:
Yep, moreover there's years that I haven't even use ISBG! :D